plugins/module_utils/foreman_helper.py

```diff
@@ -931,7 +931,7 @@ def auto_lookup_entities(self):
     def auto_lookup_nested_entities(self):
         for key, entity_spec in self.foreman_spec.items():
             if entity_spec.get('type') in {'nested_list'}:
-                for nested_key, nested_spec in self.foreman_spec[key]['foreman_spec'].items():
+                for nested_key, nested_spec in entity_spec['foreman_spec'].items():
```

Comment:
This is a no change, right?

Reply:
Correct, just something pylint spotted as being unnecessary